docs/indepth/installation.rst

```diff
@@ -374,6 +366,16 @@ Paste them into a root console on the target node:
 Post installation configuration for Fedora 20 and CentOS 7
 ----------------------------------------------------------
 
+First disable SELinux.
+
+.. task:: disable_selinux
+   :prompt: [root@node]#
+
+.. note:: Flocker does not currently set the necessary SELinux context types on the filesystem mount points that it creates on nodes.
+          This prevents Docker containers accessing those filesystems as volumes.
```

Comment:
`... containers from accessing ...` (I think omitting the "from" is a British-ism?)